src/Utilities/TrackedDependencies/FlatTrackingData.cs

```diff
@@ -943,21 +881,10 @@ public static void FinalizeTLogs(bool trackedOperationsSucceeded, ITaskItem[] re
                     }
 
                     // UNDONE: If necessary we could have two independent sets of "ignore" files, one for inputs and one for outputs
-                    // Use an anonymous method to encapsulate the contains check for the output tlogs
-                    outputs.SaveTlog(delegate (string fullTrackedPath)
-                                     {
-                                         // We need to answer the question "should fullTrackedPath be included in the TLog?"
-                                         return (!trackedFilesToRemove.ContainsKey(fullTrackedPath));
-                                     }
-                    );
-
-                    // Use an anonymous method to encapsulate the contains check for the input tlogs
-                    inputs.SaveTlog(delegate (string fullTrackedPath)
-                                     {
-                                         // We need to answer the question "should fullTrackedPath be included in the TLog?"
-                                         return (!trackedFilesToRemove.ContainsKey(fullTrackedPath));
-                                     }
-                    );
+                    // Use a anonymous methods to encapsulate the contains check for the input and output tlogs
```

Comment:
typo: `a anonymous`

Reply:
How the heck did I change it from being correct to incorrect!?

---
In reply to: [197191459](https://github.com/Microsoft/msbuild/pull/3438#discussion_r197191459) [](ancestors = 197191459)